templates/header.ejs

```diff
@@ -41,6 +41,7 @@
     <script async defer nomodule src="/assets/js/css-vars-ponyfill.js"></script>
   </head>
   <body>
+  <div>
```

Comment:
This is unfortunate :(

Reply:
i'm not sure I understand. Is it something to do with how HID works?

Reply:
I'm able to remove this extra div now that I've updated the cd.css from common_design
https://github.com/UN-OCHA/common_design/pull/188/commits/8afbd1fa721fd389d9735dcc577ecad578c18e9b

Reply:
woohoo! 🎉